README.md

```diff
@@ -1 +1,54 @@
-# proto-schema-registry
\ No newline at end of file
+# proto-schema-registry
+
+[![Action](https://github.com/syncromatics/proto-schema-registry/workflows/build/badge.svg)](https://github.com/syncromatics/proto-schema-registry/workflows/build/badge.svg)
+[![Go Report Card](https://goreportcard.com/badge/github.com/syncromatics/proto-schema-registry)](https://goreportcard.com/report/github.com/syncromatics/proto-schema-registry)
+
+Proto Schema Registry provides a service to store proto schemas in kafka. It also supports breaking change detection and versioning.
+Protobuf generated code is both backwards and forward compatible but this service provides a means for other services that don't have
+the generated protobuf objects the ability to decode and display messages.
```

Comment:
Compile-time vs. run-time is implied here, but newcomers to the project might understand the README more quickly if you called it out explicitly